x-pack/plugins/infra/public/containers/waffle/with_waffle_nodes.tsx

```diff
@@ -49,12 +51,14 @@ export const WithWaffleNodes = ({
       filterQuery,
     }}
   >
-    {({ data }) =>
+    {({ data, loading, refetch }) =>
```

Comment:
Without `notifyOnNetworkStatusChange` above, this will not rerender on `refetch`.